frontend/src/Hooks/useBikes.ts

```diff
@@ -8,10 +8,13 @@ export default function useBikes(){
     const user = useAuth(false)
     useEffect(() =>fetchBikes(), [user])
 
+    function updateBikeList(bikes: Bike[]){
+        setBikes(bikes)
+    }
```

Comment:
Why don't you directly return the setBikes function instead?

Reply:
Cause ChatGPT :-)
Exporting setBikes from the useBikes hook may not be a good practice as it can lead to unwanted side effects and make it harder to manage the state of the component that uses this hook.